bin/node/src/main.rs

```diff
@@ -105,6 +107,23 @@ fn main() -> sc_cli::Result<()> {
         Some(Subcommand::TryRuntime) => Err("TryRuntime wasn't enabled when building the node. \
         You can enable it with `--features try-runtime`."
             .into()),
+        #[cfg(feature = "runtime-benchmarks")]
```

Comment:
that's right, but I would prefer to leave this as it is, because:
 - `Subcommand::TryRuntime` is handled in the same way already
 - checking `cfg!(feature = "runtime-benchmarks")` inside increases nesting
 - when `runtime-benchmarks` is off, then `--help` doesn't display unnecessary arguments (of `BenchmarkCmd`)